package.json

```diff
@@ -15,21 +15,23 @@
   },
   "devDependencies": {
     "auto-changelog": "^1.16.1",
-    "release-it": "^10.4.4",
-    "@typescript-eslint/eslint-plugin": "3.5.0",
-    "@typescript-eslint/parser": "3.5.0",
-    "eslint": "7.15.0",
-    "eslint-config-airbnb-base": "14.2.0",
-    "eslint-config-prettier": "6.11.0",
-    "eslint-import-resolver-typescript": "2.0.0",
-    "eslint-plugin-compat": "3.8.0",
-    "eslint-plugin-import": "2.22.0",
-    "eslint-plugin-jest": "23.17.1",
-    "eslint-plugin-jsx-a11y": "6.3.1",
-    "eslint-plugin-prettier": "3.1.4",
-    "eslint-plugin-react": "7.20.2",
-    "eslint-plugin-react-hooks": "4.0.4",
-    "eslint-plugin-typescript-sort-keys": "1.2.0"
+    "release-it": "^10.4.4"
+  },
+  "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": "5.13.0",
+    "@typescript-eslint/parser": "5.13.0",
+    "eslint": "8.10.0",
```

Comment:
[`eslint 8` does not support Node 15](https://eslint.org/docs/8.0.0/user-guide/migrating-to-8.0.0#drop-old-node), but it's still the version specified in `.node-version`. [PR](https://github.com/eslint/eslint/pull/14592/files).

Since you had this running I am guessing that it is not a hard limit and we can continue using the same node version?

Nonetheless, I would recommend upgrading very shortly afterwards to not run into any issues with our tooling.

Reply:
I did not see that!